src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -309,6 +309,15 @@ private void scrape(List<MetricFamilySamples> mfs) {
 
         String unit = null;
 
+        if (rule.awsNamespace.equals("AWS/DynamoDB") && rule.awsDimensions.contains("GlobalSecondaryIndexName")) {
```

Comment:
We should only attach it on the broken metrics. This will need a quick mention in the readme too.